src/main/scala/io/qbeast/spark/internal/QbeastOptions.scala

```diff
@@ -74,17 +73,17 @@ case class QbeastOptions(
   def toMap: CaseInsensitiveMap[String] = {
     val options = Map.newBuilder[String, String]
     for (txnAppId <- txnAppId; txnVersion <- txnVersion) {
-      options += DeltaOptions.TXN_APP_ID -> txnAppId
-      options += DeltaOptions.TXN_VERSION -> txnVersion
+      options += "txnAppId" -> txnAppId
```

Comment:
Why we are not using the DeltaOptions?

Reply:
Oh, I see that `QbeastOptions` is an internal object. 
Then, I am not sure how to control the mapping between `QbeastOptions` and `DeltaOptions`. 

Should we add methods in the specific modules to verify and merge both subsets of options?